core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -880,6 +880,28 @@ export class BaseUser {
     this.page = newPage;
     return newPage;
   }
+
+  /**
+   * Scrolls to the bottom of the page.
+   */
+  async scrollToBottomOfPage(): Promise<void> {
+    await this.page.evaluate(async () => {
+      await new Promise<void>(resolve => {
+        let totalHeight = 0;
+        const distance = 100;
+        const timer = setInterval(() => {
+          const scrollHeight = document.body.scrollHeight;
+          window.scrollBy(0, distance);
+          totalHeight += distance;
+
+          if (totalHeight >= scrollHeight) {
```

Comment:
Can you add comments to describe what this function is doing (and, in particular, the signfiicance of the value 100? Maybe you can name that constant in a more specific way.)

Reply:
Okay. Changed it to a much simpler version, where the page is scrolled in one go.